packages/patternfly-4/react-docs/static/tippyOverrides.css

```diff
@@ -0,0 +1,59 @@
+/* @patternfly/react-core's tippy.js styling overrides */
```

Comment:
Should this file be copied here?

Reply:
Probably not :)

Yea I saw that, haven't gotten around to tracing it down yet. Still doing some work on this PR, will clean it up!

Reply:
Ok, so I've had a chance to remind myself about that file. Right, so basically it's being copied because there in the same way that tippy.css and tippyOverrides.css are being exposed as part of the react-core dist, for example;

<img width="212" alt="Screen Shot 2019-06-28 at 1 58 23 PM" src="https://user-images.githubusercontent.com/5942899/60361733-fbd55e80-99ac-11e9-9f09-c28708c18b5c.png">

This way, we're consuming these styles in react-docs the same way an end user would be, which is by a reference from base.css. We could do some swizzling to make this refer to the original course file for us, but not sure we want to do that? I also haven't checked out the developer experience of actually working on those styles, so I'll look into that as well to make sure there's a straight forward way to work on those styles without having to run a build every time.

In the meantime, let me know if you have a concern about exposing these tippy stylesheets like this. There are still several things left to do here so we definitely have time to talk it through if you have other ideas.